File: Python/Tarjeta.py
from Payment import Payment

class Tarjeta(Payment):
    NombreTarjeta = str
    NumeroTarjeta = str
    cvv = int
    fechaVencimiento = str

    def __init__(self, NombreTarjeta, NumeroTarjeta, cvv, fechaVencimiento):
        super.__init__(id)
        self.NombreTarjeta = NombreTarjeta
        self.NumeroTarjeta = NumeroTarjeta
        self.cvv = cvv
        self.fechaVencimiento = fechaVencimiento

File: Java/Tarjeta.java
class Tarjeta  extends Payment {
    String NombreTarjeta;
    Integer NumeroTarjeta;
    Integer cvv;
    String fechaVencimiento;

    public Tarjeta(Integer id, String NombreTarjeta, Integer NumeroTarjeta, Integer cvv, String fechaVencimiento) {
        super(id);
        this.NombreTarjeta = NombreTarjeta;
        this.NumeroTarjeta = NumeroTarjeta;
        this.cvv = cvv;
        this.fechaVencimiento = fechaVencimiento;


    }
}
